<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">3</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_file"/>
   <addaction name="action_append_from_file"/>
   <addaction name="action_save_gpx"/>
   <addaction name="action_export_fav"/>
   <addaction name="action_exit"/>
  </widget>
  <action name="action_export_fav">
   <property name="text">
    <string>Сохранить в ПроГород</string>
   </property>
  </action>
  <action name="action_save_gpx">
   <property name="text">
    <string>Сохранить в gpx</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_check_all">
   <property name="text">
    <string>Отметить все</string>
   </property>
  </action>
  <action name="action_uncheck_all">
   <property name="text">
    <string>Сбросить выбор</string>
   </property>
  </action>
  <action name="action_append_from_file">
   <property name="text">
    <string>Добавить из файла</string>
   </property>
   <property name="toolTip">
    <string>Добавить из файла</string>
   </property>
  </action>
  <action name="action_open_file">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="action_del_from_list">
   <property name="text">
    <string>Удалить из списка</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
